packages/fast-components-styles-msft/src/subheading/index.ts

```diff
@@ -0,0 +1,33 @@
+import { IDesignSystem } from "../design-system";
+import { ComponentStyles, ICSSRules } from "@microsoft/fast-jss-manager";
+import { ISubheadingClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft"
+
+function applySubheadingStyles(): ICSSRules<IDesignSystem> {
+    return {
+        fontWeight: "500"
+    };
+}
+
+const styles: ComponentStyles<ISubheadingClassNameContract, IDesignSystem> = {

```

Comment:
Instead of applying `fontWeight` to each individual level class, you should instead create a common `subheading` class that gets applied to every version of the component and then apply `fontWeight` to that.

Reply:
I arbitrarily set the fontweight normal or 400 for all, the previous setting was 500 for all and that was also arbitrary